sources/api/apiserver/src/server/ephemeral_storage.rs

```diff
@@ -118,6 +118,9 @@ pub fn bind(variant: &str, dirs: Vec<String>) -> Result<()> {
         _ => format!("{}{}", RAID_DEVICE_DIR, RAID_DEVICE_NAME),
     };
 
+    // Normalize input by trimming trailing "/"
+    dirs.iter_mut().for_each(|dir| *dir = dir.trim_end_matches("/").to_string());
```

Comment:
Nit: I think we are fine for now since the input `dirs` are only going to be used within this function. But this introduced a side effect of the input - the dirs will be modified from the original userdata. If we want to keep the input "dirs" immutable, we can introduce a local var (not a huge overhead given the size of the command args is going to me pretty small) or perform the trim in the loop.

Reply:
I think this is fine to make it mutable but we should update the docs to call out that we recommend you not include a training`/` and we'll just strip it anyway, so folks aren't surprised by the mutation.